tests/test_sagemaker/test_sagemaker_model_packages.py

```diff
@@ -133,24 +134,21 @@ def test_list_model_packages_approval_status():
 def test_list_model_packages_model_package_group_name():
     client = boto3.client("sagemaker", region_name="eu-west-1")
     group1 = "test-model-package-group"
+    client.create_model_package_group(
+        ModelPackageGroupName=group1,
+        ModelPackageGroupDescription="test-model-package-description",
+    )
     client.create_model_package(
-        ModelPackageName="test-model-package",
-        ModelPackageDescription="test-model-package-description",
         ModelPackageGroupName=group1,
     )
     client.create_model_package(
-        ModelPackageName="test-model-package",
-        ModelPackageDescription="test-model-package-description-2",
         ModelPackageGroupName=group1,
     )
     client.create_model_package(
-        ModelPackageName="test-model-package-2",
-        ModelPackageDescription="test-model-package-description-3",
         ModelPackageGroupName=group1,
     )
     client.create_model_package(
         ModelPackageName="test-model-package-without-group",
-        ModelPackageDescription="diff_group",
     )
     resp = client.list_model_packages(ModelPackageGroupName=group1)
```

Comment:
These changes are referred to the fact that you can't specify both `ModelPackageName` and `ModelPackageGroupName`. Also, we need to first create the `ModelPackageGroup` for creating its packages.